rotkehlchen/tests/api/test_nfts.py

```diff
@@ -176,3 +176,63 @@ def test_nft_balances_and_prices(rotkehlchen_api_server):
     assert expected_result == result
     assert FVal(price) > 0
     assert FVal(price_usd) > 0
+
+
+@flaky(max_runs=3, min_passes=1)  # all opensea calls have become quite flaky
+@pytest.mark.parametrize('ethereum_accounts', [[TEST_ACC4]])
```

Comment:
no no no please don't do that. This is a heavy test. Remove all opensea queries.

1. Make it so that the test starts with NFTs saved in the DB.
2. Make it so that the test has all caching satisfied so it does not query opensea again
3. Then deal with the filtering of ignores -- but the test would all be local.


Otherwise what you are doing here is crazy bad. You want to test a single sqlite DB query and you spend minutes (that's how much this test would take with opensea) to set up the DB with the same NFTs each time it runs.

Reply:
alright

Reply:
> Make it so that the test starts with NFTs saved in the DB.

This won't work for `/nfts` doesn't use any data stored in the DB. 
So I mocked the return data gotten from `_get_all_nft_data` instead.

Reply:
yeah mocking is also fine, would have to review